extensions/vertx-http/deployment/src/main/resources/dev-templates/tags/nonActionableExtension.html

```diff
@@ -2,10 +2,16 @@
 <div class="col mb-4">
     <div class="card h-100">
         <div class="card-header">
-            {it['name']}
-            {#if it['metadata'] && it['metadata']['guide']}
-            <a href="{it['metadata']['guide']}" class="text-muted float-right" title="Guide"><i class="fa fa-book"></i></a>
-            {/if}
+            <div class="row">
+                <div class="col-md-10">
```

Comment:
Why not 11?

Reply:
I don't know why, but if I put 11, the button guide ends up going down.
I think it ends up leaving the limits of the bootstrap grid.